gnomad/utils/vcf.py

```diff
@@ -436,6 +436,8 @@ def make_combo_header_text(
 
     if len(combo_dict) == 1:
         if combo_dict["group"] == "adj":
+            if "gnomad" in prefix:
+                return f" {preposition} samples in gnomAD"
```

Comment:
should we at a postfix option instead of having a check for gnomad in the prefix?

Reply:
do you mean update the ukbb code instead? this function is called in `make_info_dict`, so we'd need to add a fix after this:
```
                for label_group in faf_label_groups:
                    vcf_info_dict.update(
                        make_info_dict(
                            prefix=subset,
                            pop_names=gnomad_exomes_pops,
                            label_groups=label_group,
                            faf=True,
                        )
                    )
``` 
(also after the same call for the genomes)

Reply:
I moved the fix to `make_info_dict`, let me know what you think!